app/processing/account-codes/maps/bps.js

```diff
@@ -0,0 +1,141 @@
+const { SOS210, SOS404, SOS362, SOS351, SOS363, SOS367, SOS368, SOS369, SOS361, SOS400, SOS402, SOS260, SOS266, SOS410, SOS350, SOS364, SOS415, SOS416, SOS417, SOS420, SOS216, SOS471, SOS472, SOS473, SOS474 } = require('../../../constants/account-codes/ap')
+const { SOS330, SOS465, SOS450, SOS451, SOS452, SOS453, SOS454, SOS455, SOS456, SOS457, SOS458, SOS466, SOS267, SOS459, SOS464, SOS336, SOS460, SOS461, SOS462, SOS463, SOS217 } = require('../../../constants/account-codes/ar-admin')
+const { SOS310, SOS445, SOS430, SOS431, SOS432, SOS433, SOS434, SOS435, SOS436, SOS437, SOS438, SOS446, SOS268, SOS439, SOS444, SOS316, SOS440, SOS441, SOS442, SOS443, SOS218 } = require('../../../constants/account-codes/ar-irregular')
```

Comment:
Should these be ordered?

Reply:
I put them in order of then being used so you can see the groups of three down each line.